src/components/AddBooks/AddBook.js

```diff
@@ -3,29 +3,37 @@ import axios from "axios";
 import Navigation from "../Navigation/Navigation";
 import Grid from "../Grid/Grid";
 
-const AddBooks = () => {
+const AddBooks = (props) => {
+  const {
+    setName,
+    verified,
+    setVerified,
+    displayCart,
+    setDisplayCart,
+    signOn,
+    setSignOn,
+    employee,
+    setEmployee,
+    users,
+    setUsers,
+    search: propSearch,
+    setSearch: propSetSearch,
+    books: propBooks,
+    setBooks: propSetBooks,
+    bookCategory,
+    setBookCategory
+  } = props;
+
   const [title, setTitle] = useState("");
   const [author, setAuthor] = useState("");
   const [quantity, setQuantity] = useState("");
-  const [search, setSearch] = useState("");
-  const [books, setBooks] = useState([]);
+  const [search, setSearch] = useState(propSearch);
+  const [books, setBooks] = useState(propBooks);
```

Comment:
these are global states we don't need to override them here please delete both these lines and just use the props above